docs/users/how-to-import-plugins.md

```diff
@@ -2,58 +2,59 @@
 sidebar_position: 3
 ---
 
-# How to load model plugins
+# How to load plugins
 
-Models are developed separately to the Impact Framework core. However, the IF core developers maintain a standard library of models that can be found in this [Github repository](https://github.com/Green-Software-Foundation/if-models). We also provide some implementations of popular community models, although we rely on other members of the community to maintain them.
+Plugins are developed separately to the Impact Framework core. However, the IF core developers maintain a standard library of plugins that can be found in this [Github repository](https://github.com/Green-Software-Foundation/if-plugins). We also provide some implementations of popular community plugins, although we rely on other members of the community to maintain them.
 
-Use the following commands to install the `if-models` and `if-unofficial-models` repositories:
+Use the following commands to install the `if-plugins` and `if-unofficial-plugins` repositories:
 
 ```
-npm -i -g @grnsft/if-models
-npm -i -g @grnsft/if-unofficial-models
+npm -i -g @grnsft/if-plugins
+npm -i -g @grnsft/if-unofficial-plugins
 ```
 
-Models in these packages can then be invoked in an impl by providing their path in the model initialization, as shown in the following example:
+Plugins in these packages can then be invoked in an impl by providing their path in the plugin initialization, as shown in the following example:
 
 ```yaml
 name: if-demo
 description: demo pipeline
 tags:
 initialize:
-  models:
+  plugins:
     - name: azure-importer
```

Comment:
change 
- `- name: azure-importer` to `azure-importer:`
- `- name: cloud-instance-metadata` to `cloud-instance-metadata:`